app/javascript/components/ProductEdit/ShareTab/index.tsx

```diff
@@ -121,30 +122,36 @@ const DiscoverEligibilityPromo = () => {
   });
 
   if (!show) return null;
-
+  // TODO(review note - cleanup before merging):
+  // Before hands were only visible on mobile view but sizing was off. Now it is visible on all views.
   return (
-    <div role="status" className="promo">
-      <img src={hands} />
-      <div>
-        To appear on Gumroad Discover, make sure to meet all the{" "}
-        <a href="/help/article/79-gumroad-discover" target="_blank" rel="noreferrer">
-          eligibility criteria
-        </a>
-        , which includes making at least one successful sale and completing the Risk Review process explained in detail{" "}
-        <a href="/help/article/13-getting-paid" target="_blank" rel="noreferrer">
-          here
-        </a>
-        .
+    <Alert role="status">
+      <AlertIcon asChild>
+        <img src={hands} alt="" className="size-12 self-center" />
+      </AlertIcon>
+      <div className="flex flex-col gap-2">
+        <div>
+          To appear on Gumroad Discover, make sure to meet all the{" "}
+          <a href="/help/article/79-gumroad-discover" target="_blank" rel="noreferrer">
+            eligibility criteria
+          </a>
+          , which includes making at least one successful sale and completing the Risk Review process explained in
+          detail{" "}
+          <a href="/help/article/13-getting-paid" target="_blank" rel="noreferrer">
+            here
+          </a>
+          .
+        </div>
+        <button
+          className="w-max underline"
+          onClick={() => {
+            localStorage.setItem("showDiscoverEligibilityPromo", "false");
+            setShow(false);
+          }}
+        >
+          Close
+        </button>
       </div>
-      <button
-        className="close underline"
-        onClick={() => {
-          localStorage.setItem("showDiscoverEligibilityPromo", "false");
-          setShow(false);
-        }}
-      >
-        Close
-      </button>
-    </div>
+    </Alert>
```

Comment:
Need help with migration over here. Previosuly hands icon was only visiblie on mobile. 

This feels like a regression to me can someone help with the correct behaviour over here?

cc @laugardie 


https://github.com/user-attachments/assets/90bee046-e905-4184-bc00-d34d08bf46a7